node/node.go

```diff
@@ -894,6 +920,11 @@ func (node *AlgorandFullNode) InstallParticipationKey(partKeyBinary *[]byte) (ac
 	// Tell the AccountManager about the Participation (dupes don't matter) so we ignore the return value
 	_ = node.accountManager.AddParticipation(partkey)
```

Comment:
On closer inspection, `AddParticipation` returns a bool not an error.:
```
// The return value indicates if the key has been added (true) or    
// if this is a duplicate key (false).     
```

The code would currently overwrite the duplicate key, I think that's probably fine. In the future when we stop writing the file to begin with, it would just be a no-op.